src/card/theme.scss

```diff
@@ -45,22 +46,48 @@
 :local(.cardFooter) {
   width: 100%;
   margin-top: 2%;
+  display: flex;
+  justify-content: space-between;
   box-sizing: border-box;
   padding: 1%;
   font-size: 0.8rem;
+}
+
+:local(.card-actions) {
+  float: left;
+  align-self: center;
+  height: 100%;
+  width: 50%;
+}
 
-  span {
-    float: right;
+:local(.more-icon) {
```

Comment:
Arrow was added through css since the current icons from fontello cannot be customised to reproduce the current behaviour of the icon in card.

Reply:
ok for now.